app/views/schools/early_career_teachers/change_lead_provider/intro/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<% content_for :title, "How to change the lead provider for #{@participant.full_name}" %>
+<% content_for :before_content, govuk_back_link(href: schools_dashboard_path(school_id: @school_id)) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">How to change the lead provider for <%= @participant.full_name %></h1>
+    <p class="govuk-body">
+      You need to contact John Smith's current and new lead providers to <a href="https://www.gov.uk/guidance/how-to-manage-early-career-teacher-training#change-lead-provider" target="_blank" class="govuk-link govuk-link--no-visited-state">change the lead provider (opens in a new window or tab)</a>.
```

Comment:
`John Smith's` shouldn't be here I think

Reply:
@tonyheadford amended in https://github.com/DFE-Digital/early-careers-framework/pull/5016/commits/bcf10d399a1ba2c2ae78a9737fcccd6d16cf2950